public/usage-examples/graphics/current_clip/current-clip-1-simple-top-level.cs

```diff
@@ -0,0 +1,19 @@
+using static SplashKitSDK.SplashKit;
+using SplashKitSDK;
+        // Open a window with the title "Current Clip" and dimensions
+        Window window= OpenWindow("Current Clip", 800, 600);
+        
+        // Push a clipping area
+        Rectangle rectangle = CurrentClip();
```

Comment:
Add window in CurrentClip() -> CurrentClip(window), otherwise it is no need to create Window window.